### ISSUE TITLE
getPriceCustom returns a price when time is set before auction exists

### ISSUE THREAD

**Issue by dasanra** [gnosis/dx-price-oracle#3]
I'm creating the API endpoint that uses `getPriceCustom` and I'm testing against Rinkeby deployed version of `DutchXPriceOracle.sol`

Consider the following params:
**token**: `0x3615757011112560521536258c1e7325ae3b48ae` // Rinkeby RDN token
**time**: `1520812800` // 2018-03-12, one year ago
**whitelisted**: `true`
**maximumTimePeriod**: `388800` // I assume that Solidity uses seconds as base time measure. This is 4.5 days
**numberOfAuctions**: `9`

Regarding this I was expecting to get a price of `0,0` (no price or incomputable price). Unexpectedly I get a result:
`3.963877391197344453575983046348115674221700746820753546331534351508065746944e+75,
8.62718293348820473429344482784628181556388621521298319395315527974912e+68`

**Comment by hacker-DOM** [comment]
Very strange, the call should revert in that case
https://github.com/gnosis/dx-price-oracle/blob/e6228bb1bd7c1fb6521ae33cdb4f8ce6cb8bf215/test/tests.js#L140

Let me follow through the call on paper

**Comment by hacker-DOM** [comment]
Looks like this is an external bug.
The output I get for num is 3963877391197344453575983046348115674221700746820753546331534351508065746944. I just tried putting this into Google and a bunch of open issues came up:
https://github.com/ethereum/solidity/issues/3971#issuecomment-383756961
https://github.com/ethereum/web3.js/issues/1903#issuecomment-467191865

Looks like it's a bug in web3/ethereum-js non-transaction calls.

I also traced the call on paper:
Let's say latestAuctionIndex is 21 and let's call t_-1 the time you chose.
getPriceCustom
--> computeAuctionIndex(RDN, 1, 20, 20, t_-1)
--> computeAuctionIndex(RDN, 1, 20, 10, t_-1)
...
--> computeAuctionIndex(RDN, 1, 20, 2, t_-1)
--> recursion base case
--> clearingTime = t_1
if (t_-1 < t_1) - it is - revert("time too small")

The tests also support the fact that it will revert in such a case

As there is nothing we can do about this bug on the contract level, I am closing this issue.
